<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-7F44H94" failures="1" tests="1" name="Test(failed)" time="7.071" errors="0" timestamp="2021-04-22T15:53:32 IST">
  <testcase classname="org.testng.A" name="tc1" time="4.383">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [facebook]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [facebook]
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase classname="org.testng.A" name="tc21" time="0.0">
    <ignored/>
  </testcase> <!-- tc21 -->
  <testcase classname="org.testng.A" name="tc2" time="0.0">
    <ignored/>
  </testcase> <!-- tc2 -->
</testsuite> <!-- Test(failed) -->
